react-components/src/hooks/use3dScenes.tsx

```diff
@@ -148,6 +189,40 @@ function populateSceneMapWith360Images(
   });
 }
 
+function populateSceneMapWithGroundplanes(
+  sceneGroundPlanes: Array<NodeItem<GroundPlaneProperties>>,
+  sceneGroundPlaneEdges: Array<EdgeItem<Record<string, Record<string, Transformation3d>>>>,
+  scenesMap: Record<Space, Record<ExternalId, SceneData>>
+): void {
+  sceneGroundPlaneEdges.forEach((edge) => {
+    const { space, externalId } = edge.startNode;
+
+    const mappedGroundPlane = sceneGroundPlanes.find(
+      (groundPlane) =>
+        groundPlane.externalId === edge.endNode.externalId &&
+        groundPlane.space === edge.endNode.space
+    );
+
+    if (scenesMap[space]?.[externalId] === undefined) {
+      return;
+    }
+
+    if (mappedGroundPlane === undefined) {
+      return;
+    }
```

Comment:
Can we combine these two into one?

Reply:
Fixed in [fdb1604](https://github.com/cognitedata/reveal/pull/4181/commits/fdb16049a0672a85b1aeaec3b23762a33a8a64ce)